src/btree/bt_debug.c

```diff
@@ -1441,7 +1442,9 @@ __debug_cell(WT_DBG *ds, const WT_PAGE_HEADER *dsk, WT_CELL_UNPACK *unpack)
     case WT_CELL_VALUE_OVFL:
     case WT_CELL_VALUE_OVFL_RM:
     case WT_CELL_VALUE_SHORT:
-        WT_RET(ds->f(ds, ", ts/txn %s,%s",
+        WT_RET(ds->f(ds, ", ts/txn %s,%s,%s,%s",
```

Comment:
Just an idea, but should the string contain some information on which timestamp is which? I think start and stop are straightforward but it's less obvious when there are 4 sets of numbers.

Reply:
Yes, I added some more details to differentiate those numbers.